src/libraries/System.IO.FileSystem/tests/FileStream/CharacterDevice.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.AnyUnix)]
+    public class CharacterDevice
+    {
+        const string CharacterDevicePath = "/dev/tty";
+        [Theory]
+        [InlineData(FileOptions.None)]
+        [InlineData(FileOptions.Asynchronous)]
```

Comment:
Can we pass `devicePath` as param and test `/dev/tty`, `/dev/console` and `/dev/null`? They are available in test environments AFAICT.

Reply:
Done, I also included /dev/zero just in case.